test/http/http.go

```diff
@@ -77,9 +77,9 @@ func CaptureRoundTrip(method, scheme, hostname, path, location string) (*Capture
 					certs[i] = cert
 				}
 
-				// Verify the certificate Hostname matches the request hostname.
 				capturedTLSHostname = certs[0].DNSNames[0]
-				return certs[0].VerifyHostname(hostname)
+				// TODO: hostname verification breaks default SSL certificates
+				return nil //certs[0].VerifyHostname(hostname)
```

Comment:
I agree about not failing the test here.

Should this call to `VerifyHostname` be moved to an assertion (see state.go) and validated given the context of each scenario? 
We could assert the hostname matches if we expect it, and assert a mismatch (whatever it might be) if using an invalid host or fallback SSL cert.

Reply:
Yes, that makes sense

Reply:
done